src/pages/Wallet/WalletHomePage.tsx

```diff
@@ -130,17 +131,19 @@ const WalletHomePage = () => {
   }, [fetchData])
 
   // Polling (when pending tx)
-  useInterval(fetchData, 2000, pendingTxList.length === 0)
+  useInterval(fetchData, 2000, pendingTxList[currentNetwork]?.length === 0)
 
   if (!wallet) return null
 
+  const pendingTxLength = (pendingTxList[currentNetwork] || []).length
```

Comment:
How about instead of `pendingTxLength`, we create `pendingTxs` so we don't have to rewrite it in line `187`?
```js
const pendingTxs = pendingTxList[currentNetwork] || []
```

Then, we can also create a variable:
```js
const showSpinner = isLoading || pendingTxs.length > 0
```

and use it in lines `146` and `184`.

Reply:
Definitely. Thanks for your suggestion!